### PR TITLE
LPS-143319 Update Public footer fragment with Dialect theme.

### PR DESCRIPTION
**Pull request by rosig**

### CODE REVIEW

**Comment by rosig** [comment]
## :fire: GitRay Report 
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
 ## Jira Tasks 

 <html><details><summary>More details about: <b>LPS-143319 Update Public footer fragment with Dialect theme</b>.</summary>
              <br />
              <table>
              <tr>
            <th>Title</th>
            <td><a target="_blank" href="https://issues.liferay.com/browse/LPS-143319">Update Public footer fragment with Dialect theme</a></td>
        </tr>
              <tr>
            <th>Type</th>
            <td>Technical Task</td>
        </tr>
              <tr>
            <th>Priority</th>
            <td>Média</td>
        </tr>
              <tr>
            <th>Sub-Tasks</th>
            <td>No Sub-Tasks Opened</td>
        </tr>
              <tr>
            <th>Status</th>
            <td>In Review</td>
        </tr>
              <tr>
            <th>Assignee</th>
            <td>Rosinaldo Guedes</td>
        </tr>
              <tr>
            <th>Reporter</th>
            <td>Silas Silva</td>
        </tr>
                </table>
          </details></html> 

---

Hey, don't forget to update all related tickets with correct status, such as **In Review** :wink: 
### Git Stats 

4 files changed, 77 insertions(+), 134 deletions(-)

**Comment by silasbsi** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/public-footer/index.html [hunk 1]
```diff
@@ -1,125 +1,125 @@
-<footer class="bg-brand-primary-darken-5 footer public-footer">
+<div class="bg-brand-primary-darken-5 c-py-10 footer public-footer">
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/public-footer/index.html:1]: Regarding the semantical HTML, this should continue as **footer**.

Thinking about all Liferay users, we need to use footer tag here

image

**Review by kevenleone — CHANGES_REQUESTED** [review]
@rosig 

I noticed that in one file you changed from `div` to `footer`, which is good!

But in the other, you changed from `footer` to `div`

I think it's better to use footer in both cases, regarding semantic HTML.

https://www.w3schools.com/html/html5_semantic_elements.asp

The rest looks great!

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:reevaluate:1347401_432

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**